lib/UI.js

```diff
@@ -102,6 +102,10 @@ define(function (require, exports, module) {
 
         $("#liveDevButton").click(function () {
             IframeBrowser.detachPreview();
+
+            // Hide attached live preview pane
+            $("#second-pane").css("width", "0");
+            $("#first-pane").css("width", "100%");
```

Comment:
Two things: should we use `visibility: hidden;` vs. `width: 0`?  Also, this should be a class you apply that lives in the CSS file.  Can you also show me what it looks like when it runs?  What happens to the splitter and editor?